src/lib/ModbusRtu.cpp

```diff
@@ -94,6 +94,9 @@ void Modbus::begin(ModbusPort *pPort, unsigned long u32speed, uint16_t config)
         // return RS485 transceiver to receive mode
         pinMode(u8txenpin, OUTPUT);
         digitalWrite(u8txenpin, LOW);
+
+        pinMode(u8rxenpin, OUTPUT);
+        digitalWrite(u8rxenpin, LOW);
     }
 
```

Comment:
We are somehow missing a `Modbus::end()` function -- it's in the header file but I don't find an implementation.

Please add an `end()` function that disables both TX and RX according to the pattern, and also does an `end()` call to the data port. It should be enough to do a `Modbus::end()` to disable the com port and turn off power.

Reply:
add method `void end()` to do `Modbus::end()`.